README.md

```diff
@@ -40,12 +40,13 @@ asyncapi/generator -o ./output asyncapi.yml markdown
   Options:
 
     -V, --version                  output the version number
-    -w, --watch                    watches the template directory and the AsyncAPI document for change, and re-generate the files when they occur.
-    -o, --output <outputDir>       directory where to put the generated files (default: the current directory)
+    -w, --watch                    watches the templates directory and the AsyncAPI document for changes, and re-generate the files when they occur
+    -o, --output <outputDir>       directory where to put the generated files (defaults to current directory)
     -d, --disable-hook <hookName>  disable a specific hook
     -n, --no-overwrite <glob>      glob or path of the file(s) to skip when regenerating
     -p, --param <name=value>       additional param to pass to templates
-    -t, --templates <templateDir>  directory where templates are located (default: Internal template folder)
+    -t, --templates <templateDir>  directory where templates are located (defaults to internal templates directory)
+    --force-install                forces the installation of the template dependencies
```

Comment:
The `-f` flag will be a generic `--force` flag. It would "force" everything. Including, for instance, the generation when the repo has unstaged changes. It's useful but it's probably another flag and another PR.

Reply:
yeap, since last release we have `--force-write` so `--force-install` now nicely adds to the list and then as you wrote we can have `-f` god to apply all force flags